viewer/viewer.js

```diff
@@ -9721,7 +9721,7 @@ function processCronQueries () {
 // Main
 // ----------------------------------------------------------------------------
 function main () {
-  if (!fs.existsSync('./vueapp/dist/index.html')) {
+  if (!fs.existsSync('./vueapp/dist/index.html') && app.settings.env !== 'development') {
```

Comment:
this should use the same join dirname maybe?

Reply:
👍 good idea. done